app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, favourite_dish: str,
+                 nickname: str,
+                 hummer_level: int) -> None:
```

Comment:
There is a typo in the parameter name `hummer_level`. It should be `hammer_level` to accurately describe the attribute and maintain consistency with standard terminology.